src/test/java/org/mifos/integrationtest/cucumber/stepdef/BillPayStepDef.java

```diff
@@ -197,8 +197,8 @@ public void iShouldBeAbleToExtractResponseBodyFromCallbackForBillPay() {
                 }
                 if(rootNode != null && rootNode.has("billId") && rootNode.get("billId").asText().equals("001")) {
                     String requestId = null;
-                    if (rootNode.has("requestId")) {
-                        requestId = rootNode.get("requestId").asText();
+                    if (rootNode.has("reason")) {
```

Comment:
this looks ambiguous

Reply:
the field request id does not occur in the response body, thats why replaced with more accurate reason

Reply:
rename variable to reason